crosstool/cc_toolchain_config.bzl

```diff
@@ -1191,9 +1237,9 @@ please file an issue at https://github.com/bazelbuild/apple_support/issues/new
                     flag_group(
                         flags = [
                             "-isysroot",
-                            "%{sdk_dir}",
-                            "-F%{sdk_framework_dir}",
-                            "-F%{platform_developer_framework_dir}",
+                            "__BAZEL_XCODE_SDKROOT__",
+                            "-F{}".format(_sdk_framework_dir(platform_type, sdk_version)),
```

Comment:
FYI we use switched to using -iframework instead of -F internally, to silence some API availability warnings that were triggering in Xcode 15.0 XCTest headers.

Reply:
interesting, we haven't gotten any reports about this yet, is that surprising?

Reply:
I had to review the detail on the issue.  Apparently this doesn't happen on Xcode clang because it has some hack that suppresses diagnostics based on macro names (!).  So it shouldn't be an issue here.